src/Uno.UI/UI/Xaml/Controls/Popup/Popup.cs

```diff
@@ -311,4 +311,7 @@ internal Brush LightDismissOverlayBackground
 
 	internal static DependencyProperty LightDismissOverlayBackgroundProperty { get; } =
 		DependencyProperty.Register(nameof(LightDismissOverlayBackground), typeof(Brush), typeof(Popup), new FrameworkPropertyMetadata(defaultValue: null, propertyChangedCallback: (o, e) => ((Popup)o).ApplyLightDismissOverlayMode()));
+
+	// On WinUi, a popup is not IsTabStop but is somehow focusable. This is a workaround to match that behaviour.
+	internal override bool IsFocusableForFocusEngagement() => true;
```

Comment:
@MartinZikmund I'm completely confused at this point. In #13734, we confirmed that popups are indeed not IsTabStop. However, it seems that when nothing is focusable inside the popup, the popup ends up being focused anyway? This is necessary to make popups closable on Escape (which is blocking this PR). Note that this is the case in general, not only for popups or flyouts. Check out the  Popup_Simple sample and try to close the popups using Escape. It won't work. With this workaround, it does.